SPECS/cyrus-sasl/cyrus-sasl.spec

```diff
@@ -124,16 +248,78 @@ make %{?_smp_mflags} check
 %{_sysconfdir}/sysconfig/saslauthd
 /lib/systemd/system/saslauthd.service
 %{_libdir}/systemd/system-preset/50-saslauthd.preset
-%{_includedir}/*
-%{_libdir}/*.so*
-%{_libdir}/pkgconfig/*
-%{_libdir}/sasl2/*
 %{_sbindir}/*
-%{_mandir}/man3/*
 %{_datadir}/licenses/%{name}/LICENSE
-%{_mandir}/man8/saslauthd.8.gz
+%{_mandir}/man8/*
+
+%files devel
+%{_includedir}/*
+%{_libdir}/libsasl2.so
+%{_libdir}/pkgconfig/*.pc
+%{_mandir}/man3/*
+
+%files gs2
+%{_plugindir2}/libgs2.so
+%{_plugindir2}/libgs2.so.3*
+
+%files gssapi
+%{_plugindir2}/libgssapiv2.so
+%{_plugindir2}/libgssapiv2.so.3*
```

Comment:
I suggest using a macro for the so file version (e.g. soversion). When the version changes from 3 to 4 there will be a lot of changes otherwise.

Reply:
Good call. Let ma change that.